### PR TITLE
Additional settings endpoints

### PR DESCRIPTION
**Pull request by tomaszwarda**
Adding additional settings endpoints from issue #101

### CODE REVIEW

**Review by danielui — APPROVED** [review]

File: vmngclient/api/administration.py [hunk 1]
```diff
@@ -133,3 +143,59 @@ def enable_cloud_on_ramp_for_saas_mode(self, disable=False):
...
+    def update_controller_certificate(
+        self,
+        controller_certificate: str = "",
+        first_name: str = "",
+        last_name: str = "",
+        email: str = "",
+        validity_period: ValidityPeriod = ValidityPeriod.ONE_YEAR,
+        retrieve_interval: int = 60,
+    ) -> bool:
+        if retrieve_interval < 1 or retrieve_interval > 60:
```
Comment (igosoft) [inline vmngclient/api/administration.py:179]: Magic numbers should be moved to const named values

File: vmngclient/api/administration.py [hunk 2]
```diff
@@ -133,3 +143,59 @@ def enable_cloud_on_ramp_for_saas_mode(self, disable=False):
         data = {"mode": "on"} if not disable else {"mode": "off"}
         response = self.session.put(url_path, data)
         return True if response.status_code == 200 else False
+
+    def get_organization_name(self) -> List[str]:
+        endpoint = "/dataservice/settings/configuration/organization"
+
+        return [organizations_info.get("org") for organizations_info in self.session.get_data(endpoint)]
```
Comment (igosoft) [inline vmngclient/api/administration.py:150]: Some of these getters can be refactored . We can use enums to pick up the scope and use the same query mechanism

File: vmngclient/api/administration.py [hunk 3]
```diff
@@ -133,3 +143,59 @@ def enable_cloud_on_ramp_for_saas_mode(self, disable=False):
...
+            logger.error(f"{error.get('message')} - {error.get('details')}")
+        return True if response.status_code == 200 else False
+
+    def update_vbond_address(self, vbond_address: str, vbond_port: int) -> bool:
+        endpoint = "/dataservice/settings/configuration/device"
+        payload = {"domainIp": vbond_address, "port": vbond_port}
+
+        response = self.session.post(endpoint, json=payload)
+        logger.debug(f"vBond address set to {vbond_address}, and port set to {vbond_port}")
+        return True if response.status_code == 200 else False
```
Comment (igosoft) [inline vmngclient/api/administration.py:168]:
```suggestion
        return True if response.status_code == HTTPStatus.OK else False
```

File: vmngclient/api/administration.py (hunk 2 above)
Comment (tomaszwarda) [reply vmngclient/api/administration.py]: Implemented update, in separate PR will add getter.

**Review by danielui — DISMISSED** [review]

**Review by sbasan — APPROVED** [review]
lgtm

**Pull request merged**